ui/tests/integration/helpers/date-format-test.js

```diff
@@ -1,27 +1,37 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { dateFormat } from '../../../helpers/date-format';
```

Comment:
because this helper was moved to the core, it can no longer be accessed and rendered like it previously was.  Therefore, I pretty much rewrote the tests for this helper.  I attempted to keep in spirit with the previous test, while working with the new render flow I had to implement.